src/api_server/swagger/firecracker.yaml

```diff
@@ -808,11 +808,15 @@ definitions:
       id:
         description: MicroVM / instance ID.
         type: string
-      started:
+      state:
```

Comment:
Also replace `started` with `state` a couple of lines above in the `require` section.

Reply:
@acatangiu sure! Will work on it this week. Sorry for the delay, was busy with other items.